cli/src/main.rs

```diff
@@ -22,6 +22,23 @@ use bpflint::builtin_lints;
 use bpflint::lint;
 use bpflint::report_terminal;
 
+use bpflint::LintMatch;
+use bpflint::Point;
+use bpflint::Range;
+use std::path::Path;
+
+fn using_bpf_c_ext(path: &Path) -> bool {
```

Comment:
Just a nit I didn't notice earlier, mind renaming to `has_bpf_c_ext`? I think that is the more common nomenclature in Rust.